### PR TITLE
Homepage launch

### PR DESCRIPTION
**Pull request by alxgln**

### CODE REVIEW

File: src/components/layout/Navbar.test.tsx [hunk 1]
```diff
@@ -127,6 +127,18 @@ describe(`Navbar`, () => {
         await wrapper.getByPlaceholderText('Search for videos'),
       ).toBeVisible();
     });
+
```
Comment (harshac) [inline src/components/layout/Navbar.test.tsx:130]: When do we want to hide search?

**Review by harshac — CHANGES_REQUESTED** [review]
- The build is red.
- Also, are we using the disciplines and subjects query anywhere? If not, we should remove them as a part of this PR as well

File: src/components/layout/Navbar.test.tsx (hunk 1 above)
Comment (alxgln) [reply src/components/layout/Navbar.test.tsx:130]: On the homepage the navbar is rendered without search as it has been moved to the main page content

**Comment by edwardecook** [comment]
Merged out of band of this PR.

**Pull request closed without merging**